Dockerfiles/agent/README.md

```diff
@@ -30,3 +30,59 @@ You'll need to download one of the `datadog-agent*_amd64.deb` package in this di
 You can then build the image using `docker build -t datadog/agent:master .`
 
 To build the jmx variant, add `--build-arg WITH_JMX=true` to the build command
+
+## How to activate log collection
+
+The Datadog Agent can collect logs from containers starting at the version 6. Two installations are possible:
+ 
+- on the host: where the agent is external to the Docker environment 
+- or by deploying its containerized version in the Docker environment
+
+### Setup
+
+First let’s create two directories on the host that we will later mount on the containerized agent:
+
+- `/opt/datadog-agent/run`: to make sure we do not lose any logs from containers during restarts or network issues we store on the host the last line that was collected for each container in this directory
```

Comment:
Note that docker does not need the folders to be created beforehand (it'll create them automatically), but let's keep it step-by-step for now